auto_round/export/export_to_autoround/export.py

```diff
@@ -39,112 +41,122 @@ def check_neq_config(config, data_type, bits, group_size, sym):
     return res
 
 
-def get_autogptq_backend_config(backend, bits=4):
+def get_autogptq_qlinear(backend, bits=4, group_size=128, sym=False):
     use_triton = False
-    disable_exllamav2 = False
+    disable_exllamav2 = True
     disable_exllamav1 = False
     disable_marlin = True
     use_qigen = False
-    if backend == "gptq:qigen":
+    if "qigen" in "backend":
         use_qigen = True
-    if backend == "gptq:triton":  ##TODO refine the code
+    elif "triton" in backend:
         use_triton = True
-    if backend == "gptq:marlin":
+    elif "marlin" in backend and sym == True:
         use_triton = False
         disable_marlin = False
-    if backend == "gptq:exllamav2":  ##need v1 code to export
+    elif "exllama" in backend:  ##need v1 code to export
         use_triton = False
         disable_marlin = True
-    if backend == "gptq:exllamav1":
-        use_triton = False
-        disable_marlin = True
-    if backend == "gptq:cuda":
+    elif "cuda" in backend:
         use_triton = False
         disable_marlin = True
         disable_exllamav2 = True
         disable_exllamav1 = True
-    if bits not in [2, 4, 8]:
-        use_qigen = False
-    if bits not in [2, 4]:
-        use_triton = False
-    return use_triton, disable_exllamav1, disable_exllamav2, use_qigen, disable_marlin
-
-
-def dynamic_QuantLienar_for_packing(backend, bits, group_size):
-    if "gptq" in backend:
-        use_triton, disable_exllamav1, disable_exllamav2, use_qigen, disable_marlin = get_autogptq_backend_config(
-            backend, bits
-        )
-        from auto_gptq.utils.import_utils import dynamically_import_QuantLinear # pylint: disable=E0401
-        QuantLinear = dynamically_import_QuantLinear(
-            use_triton=use_triton,
-            desc_act=False,
-            group_size=group_size,
-            bits=bits,
-            disable_exllama=disable_exllamav1,
-            disable_exllamav2=disable_exllamav2,
-            use_qigen=use_qigen,
-            disable_marlin=disable_marlin,
-        )
-        return QuantLinear
-    ##export all use trition, inference use exllamav2
-    elif "autoround" in backend or "auto-round" in backend or "auto_round" in backend:
+
+    from auto_gptq.utils.import_utils import dynamically_import_QuantLinear  # pylint: disable=E0401
+    QuantLinear = dynamically_import_QuantLinear(
+        use_triton=use_triton,
+        desc_act=False,
+        group_size=group_size,
+        bits=bits,
+        disable_exllama=disable_exllamav1,
+        disable_exllamav2=disable_exllamav2,
+        use_qigen=use_qigen,
+        disable_marlin=disable_marlin,
+    )
+    return QuantLinear
+
+
+def dynamic_QuantLienar_for_packing(backend, bits, group_size, sym):
+    if "auto_round" in backend:
```

Comment:
autoround and auto_round are easily confused, better use replace() to remove the underscore character before if

Reply:
we will use auto_round as the only one in the futhure